package.json

```diff
@@ -33,16 +33,16 @@
     }
   },
   "dependencies": {
-    "clipboard": "2.0.1",
+    "angular": "~1.6.1",
+    "angular-aria": "~1.6.1",
+    "angular-sanitize": "~1.6.1",
+    "clipboard": "^2.0.1",
```

Comment:
Sometimes you use carets and sometimes tildes, is there any reason ?

Reply:
It depends what we want for our dependencies:

* `~` -> Get only patches
* '^` -> Get minors and patches

Reply:
I know that, what I don't know is, how do you decide what dependencies should have the patches, and what dependencies should have minor updates ? Why don't you take minor updates for Angular for example ?

Reply:
Like we do for our managers :)
Because it's only tested now for angular 1.6.x, and we don't want to update for 1.7.x